tooling/scripts/contributors.ts

```diff
@@ -14,13 +14,17 @@ async function getOcto() {
   }
 
   const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
-  const response = await octokit.request('GET /repos/{owner}/{repo}/contributors', {
-    owner: 'typehero',
-    repo: 'typehero',
+  
+  // get paginated contributors
+  const res = await octokit.paginate(octokit.rest.repos.listContributors, {
+    owner: "typehero",
+    repo: "typehero",
```

Comment:
is 100 the max

Reply:
if possible it should recursively call if there’s a next page

Reply:
AFAIK It fetches recursively, res will have all 63 (currently) contributors, even if you give page size less than 63, say 20.

Reply:
![image](https://github.com/typehero/typehero/assets/69889382/d042d162-18d0-4a42-a86f-bb29f1a8776f)